packages/gitbook-docs/src/__tests__/parseContent.test.ts

```diff
@@ -498,7 +498,7 @@ describe('parseContent', () => {
     ];
 
     const parsed = parseContent(
-      '{% swagger src="index.yaml" path="/p" method="post" %}\n[index.yaml](index.yaml)\n{% endswagger %}',
+      '{% swagger src="index.yaml" path="/p" method="post" %}{% endswagger %}',
```

Comment:
I think you should test the parser when the src attribute is empty and the API description is inside the `swagger` tag